packages/generator/ni_measurement_plugin_sdk_generator/client/__init__.py

```diff
@@ -76,6 +308,7 @@ def create_client(
     module_name: Optional[str],
     class_name: Optional[str],
     directory_out: Optional[str],
+    interactive: bool,
```

Comment:
This should be moved just above the optional parameters

Reply:
Technically, having a non-optionally field after optional field(s) is completely fine.

Example: https://github.com/ni/measurement-plugin-python/blob/09b40b03980d7208343787138ee83390e0b9610e/packages/service/ni_measurement_plugin_sdk_service/_drivers/_nidcpower.py#L30-L37

Reply:
Okay, The reasons is to sync with the same order that we have given to the click options.